src/avatar/draw.js

```diff
@@ -0,0 +1,355 @@
+        obj.material.dispose();
+
+        realityEditor.gui.threejsScene.removeFromScene(obj);
+        return cylinderMesh(startPoint, endPoint, color);
+    }
+
+    // adds a circular label with enough space for two initials, e.g. "BR" (but hides it if no initials provided)
+    function createTextLabel(objectKey, initials) {
+        let labelContainer = document.createElement('div');
+        labelContainer.id = 'avatarBeamLabelContainer_' + objectKey;
+        labelContainer.classList.add('avatarBeamLabel');
+        document.body.appendChild(labelContainer);
+
+        let label = document.createElement('div');
+        label.id = 'avatarBeamLabel_' + objectKey;
+        labelContainer.appendChild(label);
+
+        if (initials) {
+            label.innerText = initials;
+            labelContainer.classList.remove('displayNone');
+        } else {
+            label.innerText = initials;
+            labelContainer.classList.add('displayNone');
+        }
+
+        return labelContainer;
+    }
+
+    // update the laser beam text label with this name's initials
+    function updateAvatarName(objectKey, name) {
+        let matchingTextLabel = document.getElementById('avatarBeamLabel_' + objectKey);
+        if (matchingTextLabel) {
+            let initials = realityEditor.avatar.utils.getInitialsFromName(name);
+            if (initials) {
+                matchingTextLabel.innerText = initials;
+                matchingTextLabel.parentElement.classList.remove('displayNone');
+            } else {
+                matchingTextLabel.innerText = '';
+                matchingTextLabel.parentElement.classList.add('displayNone');
+            }
+        }
+    }
+
+    // when sending a beam, highlight your cursor
+    function renderCursorOverlay(isVisible, screenX, screenY, color) {
+        let overlay = document.getElementById('beamOverlay');
+        if (!overlay) {
+            overlay = document.createElement('div');
+            overlay.id = 'beamOverlay';
+            overlay.style.position = 'absolute';
+            overlay.style.left = '-10px';
+            overlay.style.top = '-10px';
+            overlay.style.pointerEvents = 'none';
+            overlay.style.width = '20px';
+            overlay.style.height = '20px';
+            overlay.style.borderRadius = '10px';
+            overlay.style.backgroundColor = color;
+            overlay.style.opacity = '0.5';
+            document.body.appendChild(overlay);
+        }
+        overlay.style.transform = 'translate3d(' + screenX + 'px, ' + screenY + 'px, 1201px)';
+        overlay.style.display = isVisible ? 'inline' : 'none';
+    }
+
+    // Shows an "Establishing Connection..." --> "Connected!" label in the top left
+    function renderConnectionFeedback(isConnected) {
+        if (!statusUI) {
+            statusUI = document.createElement('div');
+            statusUI.id = 'avatarStatus';
+            statusUI.style.pointerEvents = 'none';
```

Comment:
All of these rules except for the dynamic `top` could be put in a css rule to keep things a bit more cleanly separated